src/utils/hydrate-config.ts

```diff
@@ -0,0 +1,10 @@
+import R from 'ramda'
+import { substituteWithEnv } from './substitute-with-env'
+
+export const hydrateConfig = (runtimeEnvironment: string) =>
```

Comment:
Forgive my ramda illiteracy, what does this do exactly?

Reply:
1. It replaces all occurrences of `${VAR}` with `process.env[VAR]`. The JSON-part is to facilitate the replacement so the problem is as simple as replacing captured regex groups in a string.
2. It adds the field `runtimeEnvironment` to the config object.